frontend/src/scenes/session-recordings/player/controller/PlayerSeekbarPreview.tsx

```diff
@@ -66,11 +70,28 @@ const PlayerSeekbarPreviewFrame = ({
     )
 }
 
-function _PlayerSeekbarPreview({ minMs, maxMs, seekBarRef, activeMs }: PlayerSeekbarPreviewProps): JSX.Element {
+function _PlayerSeekbarPreview({
+    minMs,
+    maxMs,
+    seekBarRef,
+    activeMs,
+    timestampFormat,
+    startTime,
+}: PlayerSeekbarPreviewProps): JSX.Element {
     const [percentage, setPercentage] = useState<number>(0)
     const ref = useRef<HTMLDivElement>(null)
     const fixedUnits = maxMs / 1000 > 3600 ? 3 : 2
-    const content = colonDelimitedDuration(minMs / 1000 + ((maxMs - minMs) / 1000) * percentage, fixedUnits)
+
+    const progressionSeconds = ((maxMs - minMs) / 1000) * percentage
```

Comment:
ah, so this calculation is duplicated?
feels like there's a lot of embedded knowledge in it 🙈 
could we extract a pure function so that at least the two places that need to know end up asking for info from a third?

Reply:
That's actually the unique bit that I _think_ Ben wrote.

The shared part is https://github.com/PostHog/posthog/pull/24837/files#diff-6224a6fa7cc05c6010397350779b8c9af4dcfe1c89c6d9bbf82f044858cb3270R90-R94 which I guess could be somewhat but even so not exactly as is (formatting is shorter for example). I'm inclined to leave it until we know we need it in a third